src/js/lib/helpers.js

```diff
@@ -290,4 +291,78 @@
 		}
 	};
 
+	/**
+	 * Finds the root domain
+	 */
+	object.findRootDomain = function () {
+		var cookiePrefix = '_sp_root_domain_test_';
+		var cookieName = cookiePrefix + new Date().getTime();
+		var cookieValue = '_test_value_' + new Date().getTime();
+
+		var split = window.location.hostname.split('.');
+		var position = split.length - 1;
+		while (position >= 0) {
+			var currentDomain = split.slice(position, split.length).join('.');
+			cookie.cookie(cookieName, cookieValue, 0, '/', currentDomain);
```

Comment:
According to the documentation: https://github.com/litejs/browser-cookie-lite

"If set to 0, or omitted, the cookie will expire at the end of the session (when the browser closes)."